pkg/component/kubernetes/apiserverexposure/sni.go

```diff
@@ -86,12 +86,11 @@ func init() {
 
 // SNIValues configure the kube-apiserver service SNI.
 type SNIValues struct {
-	Hosts               []string
-	APIServerProxy      *APIServerProxy
-	IstioIngressGateway IstioIngressGateway
-	IstioTLSTermination bool
-	WildcardHost        *string
-	WildcardTLSSecret   *corev1.Secret
+	Hosts                 []string
+	APIServerProxy        *APIServerProxy
+	IstioIngressGateway   IstioIngressGateway
```

Comment:
Can the change work without `IstioIngressGateway` here? it is already being passed in `WildcardConfiguration`.